### ISSUE TITLE
Generic Confd Query

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#620]
Originally reported of Google Code with ID 564.

```
Currently Confd replies to a given set of queries. Most of them are providing some aggregate information, and therefore they should be kept as they are (and, where needed, expanded). But sometimes it is useful to just get simple information contained in the config.data file.

A generic query, allowing the user to ask for a specific, single, piece of information, should be added.
```
Originally added on 2013-09-04 11:35:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Move all issues assigned to 2.10 to 2.11, because their priority would not justify an inclusion in 2.10.
```
Originally added on 2014-02-19 10:07:59 +0000 UTC.

Added to Milestone: Release2.11
Removed from Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
The request "ReqConfigQuery" allows a generic config query with a path (RFC6901: JSON path query). There is a tool query-config that can be used for manual queries.
```
Originally added on 2014-10-01 10:48:15 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.13
Removed from Milestone: Release2.11

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:02 +0000 UTC.

Changed State: Released
